Pack-Smart/pack-smart-fe
# Style cleanup

## What's this PR do?
- Adds aria labels and outlines to buttons
- Remove negative margin on `Home`
- Fix gap between background on `SavedView` and `HowItWorks`
- Fix bug that kept error message showing on quiz after a user resumed typing
- Fix error message that was wider than mobile view

## Where should the reviewer start?
- See aria-labels in `MultipleChoice`, `Hamburger`, `Header`
- Scss for `Home`, `SavedView`, `HowItWorks`

## How should this be manually tested?
- test in browser and in mobile view. Make sure background extends to bottom of page. 
- Run lighthouse test.
 
## Any background context you want to provide?
- No.

## What are the relevant tickets?
- Closes #21

## Screenshots/GIFs
- 
<img width="794" alt="Screen Shot 2021-03-03 at 8 20 14 PM" src="https://user-images.githubusercontent.com/68157759/109895811-e32b4a80-7c5d-11eb-96ae-7b0d1ecca84f.png">


## Questions:
- no

## Did you update the README?
- no
